### PR TITLE
Add servicebuilder blade tools template

### PR DESCRIPTION
**Pull request by jrao**
Add servicebuilder blade tools template; creates a project containing three modules using a multi-project Gradle build configuration; takes project name argument and a package path argument; the package path argument specifies the root package in which service builder classes are to be generated

### CODE REVIEW

File: com.liferay.blade.eclipse.provider/src/com/liferay/blade/eclipse/provider/cmds/CreateProjectCommand.java [hunk 1]
```diff
@@ -120,6 +120,26 @@ public Object createProject(
...
+		
+		if (ProjectType.servicebuilder.equals(type)) {
+			if (packageName.isEmpty()) {
+				return
+					"if type is servicebuilder, the name of the root package" +
+					"within which to create service builder classes must be" +
+					"specified.";
+			}
+
+			subs.put("_pkg_", packageName);
```
Comment (gamerson) [inline com.liferay.blade.eclipse.provider/src/com/liferay/blade/eclipse/provider/cmds/CreateProjectCommand.java]: We already have a substitute for package name up above, lets just use that one and if we need to, set it to the value of the argument if it is passed in.

subs.put("_package_", name.toLowerCase().replaceAll("-", "."))

File: com.liferay.blade.eclipse.provider/templates/gradle/servicebuilder/_api_/bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Bundle-Version: 1.0.0.${tstamp}
+
+Import-Package: \
+    com.liferay.portal.service;version="7.0.0",\
+    org.osgi.framework;version="[1.7,2)",\
+    *
+
+Export-Package: \
+	_pkg_.exception,\
```
Comment (gamerson) [inline com.liferay.blade.eclipse.provider/templates/gradle/servicebuilder/_api_/bnd.bnd]: Use _package_ instead

File: com.liferay.blade.eclipse.provider/templates/gradle/servicebuilder/_svc_/build.gradle [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+		classpath 'biz.aQute.bnd:biz.aQute.bnd.gradle:3.0.0'
+		classpath group: "com.liferay", name: "com.liferay.gradle.plugins.service.builder", version: "1.0.5"
+	}
+}
+
+apply plugin: 'biz.aQute.bnd.builder'
+apply plugin: "com.liferay.portal.tools.service.builder"
+
+repositories {
+	mavenCentral()
```
Comment (gamerson) [inline com.liferay.blade.eclipse.provider/templates/gradle/servicebuilder/_svc_/build.gradle]: Lets put mavenCentral below local

**Comment by gamerson** [comment]
Closing see comments and resend, thanks!

**Comment by jrao** [comment]
Resent: https://github.com/gamerson/liferay-blade-tools/pull/16

**Pull request closed without merging**